### PR TITLE
Add better logging to PromMetrics and refactor PromMetricsLive

### PR DESCRIPTION
**Pull request by 4n4nd**
Added detailed logs for metric packets downloaded in PromMetrics.
Refactored PromMetricsLive to use PromMetrics so any new changes to PromMetrics will be reflected in PromMetricsLive as well.

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -10,12 +10,18 @@ def main():
...
-                                    metric_start_datetime=metric_start_datetime,
-                                    metric_end_datetime=metric_end_datetime,
-                                    metric_chunk_size=metric_chunk_size) # this is an observable that streams in all the data alerts->etcd->build
+        metric_chunk_size = os.getenv("FLT_METRIC_CHUNK_SIZE","15m")
+
+        if os.getenv("FLT_LIVE_METRIC_COLLECT","False") == "True":
+            print("Live Metrics Collection Mode")
+            metrics_observable = metrics.PromMetricsLive(metrics_list=metrics_list,
+                                                        metric_chunk_size=metric_chunk_size) # this is an observable that streams in all the data alerts->etcd->build
+            pass
```
Comment (durandom) [inline app.py]: why `pass` here?

File: app.py [hunk 2]
```diff
@@ -74,6 +80,10 @@ def add_scores(value):
     # connect the metrics stream to publish data
     metrics_observable.connect()
 
+    if os.getenv("FLT_LIVE_METRIC_COLLECT","False") == "True":
+        sleep(60*20) # Run live metric collection and analysis for 20 minutes
```
Comment (durandom) [inline app.py]: I guess this to keep the collector running. But why only for 20 minutes? Let's just keep it running forever.

File: metrics/prommetrics.py [hunk 3]
```diff
@@ -2,12 +2,18 @@
...
 
 from rx import Observable
 
 #Prometheus connection stuff
 from prometheus import Prometheus
 
+# Set up logging
+_LOGGER = logging.getLogger(__name__)
+# Log record format
+logging.basicConfig(format='%(asctime)s:%(levelname)s: %(message)s', level=logging.INFO)
```
Comment (durandom) [inline metrics/prommetrics.py:15]: lets setup the logger in `app.py` like in https://github.com/AICoE/log-anomaly-detector/commit/c0328cc940dc2de1ae53d66ab0bbe89b0623adb5

**Review by durandom — COMMENTED** [review]
thanks for proper logging. 
some nitpicks but otherwise good

File: app.py (hunk 2 above)
Comment (MichaelClifford) [inline app.py]: Does this mean we will only get updated scores every 20 minutes?

**Review by MichaelClifford — APPROVED** [review]
LGTM

File: app.py (hunk 2 above)
Comment (4n4nd) [reply app.py]: If I keep a while (1) loop here, then I don't know how to end the program gracefully if its running on openshift

**Pull request merged**